src/Microsoft.ML.TensorFlow/TensorFlow/TensorGeneric.cs

```diff
@@ -1,5 +1,6 @@
-// Copyright (c) Microsoft. All rights reserved.
```

Comment:
This file appears to be generated from:

https://github.com/dotnet/machinelearning/blob/fad2b23f1c14082ba03c2ad1f41ccc0f90ae5110/src/Microsoft.ML.TensorFlow/TensorFlow/TensorGeneric.tt#L8-L9

you'll want to update the .tt file so the next time someone wants to edit this code, they don't regress this change.

Reply:
very good catch! May be I should run the tool on tt files too

Reply:
Yes, I see that the license comment is after some headers in .tt. May be you can put the license on top and then your tool will do the job on .tt too.

---
In reply to: [280540275](https://github.com/dotnet/machinelearning/pull/3651#discussion_r280540275) [](ancestors = 280540275)